src/main/java/duke/command/Command.java

```diff
@@ -1,5 +1,13 @@
 package duke.command;
 
+/**
+ * Abstract class for Commands in Duke application
+ */
 public abstract class Command {
```

Comment:
I concur with the call to make Command an abstract class for all your commands to extend from. You could consider adding your "run" function in this parent class as an abstract method to be implemented. This might remind all future programmers to ensure this method is there for their command to work.